frontend/src/components/ViewTrips/trips-container.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import Trip from './trip';
```

Comment:
Is this a specific file? In that case, it should be './trip.js'.

Reply:
Ah yes, missed that one. Fixed.